python/py-ezdxf/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup python        1.0
```

Comment:
the "python" part should be aligned with the rest of the Portfile:
```
PortSystem              python 1.0
```

Reply:
Done.

For potential future submissions : Should I then align the version numbers of multiple port groups (not like in this case) in a third column? For instance, like this:
```
PortGroup             cmake  1.0
PortGroup             github 1.0
```
Or should there always be one space in between port group and its version?

Reply:
One space is fine.